App.tsx

```diff
@@ -12,34 +14,81 @@ import {
   View,
 } from 'react-native';
 
-import {
-  Colors,
-  DebugInstructions,
-  Header,
-  LearnMoreLinks,
-  ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-
+import Icon  from 'react-native-vector-icons/Ionicons';
 
+import {colors,screenWidth} from './src/theme'
 
 import HomeScreen from './src/screens/homeScreen';
 import Details from './src/screens/details';
 import Cart from './src/screens/cart';
 import ItemDetail from './src/screens/itemDetail';
+import Splash from './src/screens/splash';
+import Login from './src/screens/login';
+import SignUp from './src/screens/signup'
 
 const Stack = createNativeStackNavigator();
+const Tab = createBottomTabNavigator();
+
+function home(){
```

Comment:
Create separate files for navigation containers ( Dashboard, Auth ) and use App.tsx file as a wrapper only.